### ISSUE TITLE
Problem with building on CentOS Linux release 7.3.1611 (Core) : json-c not found

### ISSUE THREAD

**Issue by ni4** [riboseinc/rnp#197]
Building in docker container on macOS 10.12.x fails with json-c not found error.

The problem is caused by wrong pkg-config search paths:
```
[root@70b1c8f26e08 rnp]# pkg-config --variable pc_path pkg-config
/usr/lib64/pkgconfig:/usr/share/pkgconfig
```
However json-c installs it's .pc file into /usr/local/lib/pkgconfig or /usr/lib/pkgconfig (if installed with --suffix=/).

Doing the following fixes the issue, however not sure where it is better to check this/set this:
`export PKG_CONFIG_PATH=/usr/lib/pkgconfig:/usr/local/lib/pkgconfig
`

**Comment by frank-trampe** [comment]
That's odd. I built on CentOS 7.3.1611 and did not experience this problem. I had to install `json-c-devel` manually, though.

**Comment by ni4** [comment]
I cloned master, maybe this is the reason. Will re-check tomorrow.

**Comment by ronaldtse** [comment]
The easiest way to setup on a mac is to run:
```
./docker.sh
```

Then inside the container:
```
./build-install.sh
```
Which will install all dependencies.

**Comment by ronaldtse** [comment]
Actually the `prepare_build.sh` script is missing JSON-C.

You can install json-c with this:
```sh
rpm --import https://github.com/riboseinc/yum/raw/master/ribose-packages.pub
curl -L https://github.com/riboseinc/yum/raw/master/ribose.repo > /etc/yum.repos.d/ribose.repo
yum install -y json-c12
```

**Comment by ronaldtse** [comment]
This issue is resolved in #212.

**Comment by ronaldtse** [comment]
Closed by merging of #212.
